src/main/java/Exercise12.java

```diff
@@ -0,0 +1,17 @@
+public class Exercise12 {
+    public int calculateFactorial(int number) {
+        int result = 1;
+        for (int i = 2; i <= number; i++) {
+            result *= i;
+        }
+        return result;
+    }
+
+    public double calculateSum(int number) {
+        double result = 0;
+        for (int i = 0; i < number; i++) {
+            result += 1 / calculateFactorial(i);
```

Comment:
Tránh việc tính giai thừa bằng công thức này 
![Screen Shot 2019-05-15 at 11 45 51 PM](https://user-images.githubusercontent.com/10450325/57824251-ac541e00-77c4-11e9-8854-11d247d52727.png)

Reply:
Em sửa rồi ạ.

Reply:
That's sửa rồi 👎

Reply:
Dạ em sửa rồi anh .